net/modemmanager/files/25-modemmanager-usb

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+# Copyright (C) 2019 Aleksander Morgado <aleksander@aleksander.es>
+
+# We need to process only full USB device removal events, we don't
+# want to process specific interface removal events.
+[ "$ACTION" = remove ] || exit
+[ -z "${INTERFACE}" ] || exit
+
+# Load common utilities
+. /etc/modemmanager/modemmanager.common
```

Comment:
Move this to `/usr/share` - it is code, not configuration

Reply:
Done in a follow up commit.